_content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Years That The Days and Months Turned Into"
+authors:
+  - "Shafer Hall"
+subcat: poetry
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1L1cUFxeqCq2FTsAsVSCSmbrHGTrHo9dW/view?usp=sharing"
+formats: [mp3]
+course: thought
+tags:
+  - imagination
+  - contemporary-poetry
+year: 2022
+month: may
+publisher: "American Public Media"
+series: slowdown
+number: 680
+minutes: 5
+---
+
+> I was the angry one, and I was the sad one,
```

Comment:
Subtle, but in markdown, lines of poetry must end in two spaces to mark the line break as intentional.